README.md

```diff
@@ -86,6 +86,18 @@ Some users may wish to exclude Fivetran error and warnings messages from the fin
 vars:
     fivetran_log_using_sync_alert_messages: false # this will disable only the sync alert messages within the connector status model
 ```
+
+### Specifying Usage Cost vs Credits Used Sources
+This package takes into consideration that the Fivetran pricing model has adjusted over the years. In particular, this package considers the old credit model (captured within the `credits_used` source) and the newer usage model (captured within the `usage_cost` source). By default, this package will dynamically check the mentioned sources in your destination and generate the respective staging models if the source is found. If the source is not found, the package will create a single row of null records in order to imitate the schema structure and ensure downstream transformations succeed. See the [does_table_exist()](fivetran_log__usage_pricing) macro for more information on this dynamic functionality.
+
+If for any reason you want to turn off the dynamic functionality of the package and use only one of the two sources (`usage_cost` or `credits_used`) you may use the respective below variables to control the package behavior. Please note, it is highly encouraged to ensure at least one of the variables is defined to `true`. The package is intended to work when both are `true` or when at least one is `true`.
+
+```yml
+vars:
+  fivetran_log__usage_pricing: true ## Dynamically checks the source at runtime to set as either true or false. May be overridden using this variable if desired.
+  fivetran_log__credits_pricing: true ## Dynamically checks the source at runtime to set as either true or false. May be overridden using this variable if desired.
```

Comment:
should the example code be setting these variables to `False`, since we're describing how to turn this behavior off?

Reply:
also the empty table still gets created if you set the variable to false. is that intended? if so, i feel like we should clarify that here, as it kinda sounds like these variables are supposed to totally **disable** the respective staging models/transformations

Reply:
Yeah I was going back and forth on this as well. We originally did not document this at all because of this functionality of it not _really_ being disabled. So I tried to add context around this.

Let me try and clarify a bit more. I also didn't want to show both of them set to `false` since that is the one form of the package that is not supported.

Reply:
@fivetran-jamie I just attempted to update the wording. I am still not 100% on the wording. What are your thoughts on the latest version?

Reply:
i guess the behavior itself is a little confusing -- why would a customer explicitly elect to bring in null data? i get that if they no longer use credits they don't want any credit-related data in there, but ideally these variables would remove any credit columns or transformations

but if that's out of scope for rn, i think we can frame the point of these variables better. like "if your schema contains outdated tables reflecting old price plans that you would like the package to completely ignore, set these to false"

Reply:
Yeah the tricky part here is that the credit model may still be applicable for some folks. There very well may be cases where people want both credit and usage cost. However, I really don't think there will be cases when people want to actively disable the models for the results of the output. The variables are probably more important in case the dynamic functionality doesn't function as intended.

As such, I wonder if it is better for this to be framed more as an informative section around the existence of the variables and less about how a customer would use them (since that is not really the intended outcome).

Reply:
i like it!